eslint.config.js

```diff
@@ -0,0 +1,39 @@
+// eslint.config.js
+const { defineConfig } = require('eslint-define-config');
+
+// ESLint recommended rules directly defined
```

Comment:
Not quite sure where this has come from, but these aren't ESLint recommended rules, they're the custom overrides the legacy config was doing _on top_ of the recommended rules.